VideoLocker/res/drawable/rectangle_with_bottom_border.xml

```diff
@@ -1,13 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-    <item>
-        <shape android:shape="rectangle">
-            <solid android:color="@android:color/darker_gray" />
-        </shape>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true">
```

Comment:
OK. In that case, this can be simplified to move the `selector` inside the second layer of the `layer-list`, as that is the only part that is dynamic.

Also, I see some other drawables similar to this applied on other areas. Should these be converted into selectors as well?